### PR TITLE
updating terraform to 0.12.15

### PR DESCRIPTION
**Pull request by CosminBriscaru**

### CODE REVIEW

File: kube.tf [hunk 1]
```diff
@@ -35,16 +35,16 @@ resource "google_container_cluster" "kube" {
   }
 
   lifecycle {
-    ignore_changes = ["node_pool", "master_auth"]
+    ignore_changes = [node_pool, master_auth]
```
Comment (cip9) [inline kube.tf]: I don’t think this is ok, but maybe I am wrong

File: mysql.tf [hunk 2]
```diff
@@ -75,16 +75,16 @@ resource "google_sql_database_instance" "mysql_db" {
   }
 
   lifecycle {
-    ignore_changes = ["project", "region"]
+    ignore_changes = [project, region]
```
Comment (cip9) [inline mysql.tf]: Same here

File: postgres.tf [hunk 3]
```diff
@@ -75,16 +75,16 @@ resource "google_sql_database_instance" "postgres_db" {
   }
 
   lifecycle {
-    ignore_changes = ["project", "region"]
+    ignore_changes = [project, region]
```
Comment (cip9) [inline postgres.tf]: Same here

File: kube.tf (hunk 1 above)
Comment (CosminBriscaru) [reply kube.tf]: Quoted references are deprecated: In the depends_on and ignore_changes meta-arguments, quoted references like "aws_instance.foo" should be rewritten without the quotes, e.g. as aws_instance.foo

File: mysql.tf (hunk 2 above)
Comment (CosminBriscaru) [reply mysql.tf]: Quoted references are deprecated: In the depends_on and ignore_changes meta-arguments, quoted references like "aws_instance.foo" should be rewritten without the quotes, e.g. as aws_instance.foo

**Pull request merged**